src/aat/resources/features/common/case/Caseworker1_Full_Case.td.json

```diff
@@ -0,0 +1,22 @@
+{
+  "_guid_": "Caseworker1_Full_Case",
```

Comment:
could this file extend the Standard_Full_Case_Creation_Data file? That would mean you could extend the Standard_Full_Case_Data file in some of your other test data files to avoid having lots of duplication of the case data

Reply:
it would also make the file name (Full_Case) more realistic, as it is not a full case at the moment

Reply:
Moved the file to F-042 folder and renamed it to "Case-Creation-using-Caseworker1-Role".